command/pr.go

```diff
@@ -560,25 +562,45 @@ func prMerge(cmd *cobra.Command, args []string) error {
 			return err
 		}
 
-		if currentBranch == pr.HeadRefName {
-			err = git.CheckoutBranch(repo.DefaultBranchRef.Name)
-			if err != nil {
-				return err
+		branchSwitchString := ""
+
+		if deleteLocalBranch {
+			var branchToSwitchTo string
+			if currentBranch == pr.HeadRefName {
+				branchToSwitchTo = repo.DefaultBranchRef.Name
+				err = git.CheckoutBranch(repo.DefaultBranchRef.Name)
+				if err != nil {
+					return err
+				}
+			}
+
+			localBranchExists := git.HasLocalBranch(pr.HeadRefName)
+			if localBranchExists {
+				err = git.DeleteLocalBranch(pr.HeadRefName)
+				if err != nil {
+					err = fmt.Errorf("failed to delete local branch %s: %w", utils.Cyan(pr.HeadRefName), err)
+					return err
+				}
+			}
+
+			if branchToSwitchTo != "" {
+				branchSwitchString = fmt.Sprintf(" and switched to branch %s", utils.Cyan(branchToSwitchTo))
 			}
 		}
 
-		err = git.DeleteLocalBranch(pr.HeadRefName)
+		err = api.BranchDeleteRemote(apiClient, baseRepo, pr.HeadRefName)
```

Comment:
> this PR comes from a fork, we probably don't want to attempt to delete a same-named branch in the base repo since that won't be the same branch. 

👍 I've added a cross-repo PR check to prevent branch deletion on PRs that come from forks.

> We could change the order of the logic: delete the remote branch before attempting to delete the local branch. 

I'm going to skip this for now because I want to reduce the number of changes before we release this.